packages/js/components/src/experimental-select-control/utils.ts

```diff
@@ -1,20 +1,23 @@
-/**
- * Internal dependencies
- */
-import { ItemType, SelectedType } from './types';
-
-export const itemToString = ( item: ItemType | null ) => {
+import { getItemLabelType } from './types';
+export const defaultGetItemLabel = < ItemType >( item: ItemType | null ) => {
 	return item ? item.label : '';
```

Comment:
Referencing `item.label` and `item.value` below are causing TS errors, since TS isn't aware that they're only intended to be used along with the default item type. Is there any way to remedy this in a TS supported way?

Reply:
Yeah, you can make us of the `is` type guard -> https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards
The easiest way to do this is by creating a `isDefaultItemType` function, like so:
```
function isDefaultItemType(item: ItemType | null): item is ItemType {
  return (item as ItemType).label !== undefined;
}
```
You could include the check for `value` here as well if you wanted.
Then in the above code you can do:
```
if ( isDefaultItemType(item) ) {
  return item.label;
} else {
return '';
}
```
TypeScript should not complain about `item.label` within the if statement now.